css/index.css

```diff
@@ -70,6 +70,29 @@ p {
     width: 800px;
     margin: 0 auto;
 }
+header {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 2.5% 0 2.7%;
+}
+header figure.logo{
+    justify-content: flex-start;
+}
+header nav {
+    width: 80%;
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-end;
+    font-size: 0.5rem;
+}
```

Comment:
You can also add a `header nav a { ... }` selector block to change the font-size of the a tags.

Reply:
i'll do that, thanks for all of your comments!